src/Microsoft.VisualStudio.ProjectSystem.FSharp/ProjectSystem/LanguageServices/FSharpParseBuildOptions.cs

```diff
@@ -90,7 +90,7 @@ public void HandleCommandLineNotifications(string binPath, BuildOptions added, B
         {
             if (added is FSharpBuildOptions fscAdded)
             {
-                foreach (var handler in _handlers)
+                foreach (Action<string, ImmutableArray<CommandLineSourceFile>, ImmutableArray<CommandLineReference>, ImmutableArray<string>> handler in _handlers)
```

Comment:
*sigh*

Reply:
It's just showing you how complex this signature is; in this case we should used interfaces.